src/magento/magento-marketplace-install-extension.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Install an Extension
+group: getting-started
+---
+
+{: .bs-callout-info}
+If your Magento installation is hosted on Adobe Commerce Cloud, see [Install, manage, and upgrade extensions][1].
```

Comment:
This link to devdocs is broken in the staging build.

Reply:
Fixed.